### PR TITLE
Installing gems

### PR DESCRIPTION
**Pull request by dqnamo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -34,6 +34,15 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'haml-rails', '~> 2.0'
+gem 'autoprefixer-rails'
+gem 'high_voltage', '~> 3.1'
+
+#gem 'activeadmin'
+
```
Comment (danielpaul) [inline Gemfile:43]:
```suggestion

# Performance
```

File: Gemfile [hunk 2]
```diff
@@ -34,6 +34,15 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'haml-rails', '~> 2.0'
```
Comment (danielpaul) [inline Gemfile:38]:
```suggestion
# Front end
gem 'haml-rails', '~> 2.0'
```

File: Gemfile [hunk 3]
```diff
@@ -46,6 +55,16 @@ group :development do
...
   gem 'spring-watcher-listen', '~> 2.0.0'
+
+  gem 'letter_opener'
+  gem 'annotate'
+  gem "better_errors"
+  gem "binding_of_caller"
+
+  gem 'bullet'
+  gem 'rubocop'
+  gem 'paranoia', '~> 2.2'
```
Comment (danielpaul) [inline Gemfile]: this needs to be moved to all group as this will be used in production for soft deleting records

File: Gemfile [hunk 4]
```diff
@@ -46,6 +55,16 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+
+  gem 'letter_opener'
+  gem 'annotate'
+  gem "better_errors"
+  gem "binding_of_caller"
```
Comment (danielpaul) [inline Gemfile:67]: why do we need this gem?

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: Gemfile [hunk 5]
```diff
@@ -34,6 +34,15 @@ gem 'jbuilder', '~> 2.5'
...
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'haml-rails', '~> 2.0'
+gem 'autoprefixer-rails'
+gem 'high_voltage', '~> 3.1'
+
+#gem 'activeadmin'
+
+gem 'rack-timeout'
+gem 'sidekiq'
```
Comment (danielpaul) [inline Gemfile:46]: add sidekiq.yml config file as well

File: Gemfile (hunk 4 above)
Comment (dqnamo) [reply Gemfile:67]: It gives us some additional error information.

**Review by danielpaul — CHANGES_REQUESTED** [review]

**Pull request merged**